bin/jupyter-dashboard-admin

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env node
+/**
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+
+/**
+ * Command-line script which allows clearing of the Dashboard Server cache.
+ */
+
+var fs = require('fs');
+var hjson = require('hjson');
+var nconf = require('nconf');
+var path = require('path');
+var request = require('request');
+var urljoin = require('url-join');
+var yargs = require('yargs');
+
+// Config defaults are in an HJSON file in the root of the source tree
+var defaultConfig = path.join(__dirname, '..', 'config.json');
+var config = nconf.file({ file: defaultConfig, format: hjson });
+
+var DEFAULT_HOST = 'http://localhost:3000';
```

Comment:
First, don't want to include **config.js**, since that handles its own `nconf` instance and does other things which we don't want to do in this CLI app.

I could read in and parse `PUBLIC_LINK_PATTERN`, replacing `{host}` & `{port}` with `IP` & `PORT` config options. Default to `http` for `{protocol}`.